content/features/featuresDeepDive/environment/environment_introduction.md

```diff
@@ -166,3 +94,9 @@ scene.fogColor = new BABYLON.Color3(0.9, 0.9, 0.85);
 See, we told you it was easy.
 
 If you want to see and play with the playground scene for this tutorial, you can check it out here: <Playground id="#7G0IQW" title="Simple Fog Example" description="Simple example of how to add fog to your scene." image="/img/playgroundsAndNMEs/divingDeeperEnvironmentIntro2.jpg" isMain={true} category="Environment"/>
+
+### Skybox
+
+To give a perfect illusion of a beautiful sunny sky, we are going to create a simple box, but with a special texture to simulate a sky.
+
+All the information regarding this skybox creation can be found [here](/features/featuresDeepDive/environment/skybox).

```

Comment:
it is preferable not to have "here" as a link title. maybe something like "in the skybox documentation"?

```suggestion
All the information regarding this skybox creation can be found in the [skybox documentation](/features/featuresDeepDive/environment/skybox).
```